tooling/nargo_cli/src/cli/compile_cmd.rs

```diff
@@ -263,23 +257,23 @@ fn save_program(
 fn save_contract(contract: CompiledContract, package: &Package, circuit_dir: &Path) {
     // TODO(#1389): I wonder if it is incorrect for nargo-core to know anything about contracts.
     // As can be seen here, It seems like a leaky abstraction where ContractFunctions (essentially CompiledPrograms)
-    // are compiled via nargo-core and then the PreprocessedContract is constructed here.
+    // are compiled via nargo-core and then the ContractArtifact is constructed here.
     // This is due to EACH function needing it's own CRS, PKey, and VKey from the backend.
     let debug_artifact = DebugArtifact {
         debug_symbols: contract.functions.iter().map(|function| function.debug.clone()).collect(),
         file_map: contract.file_map,
         warnings: contract.warnings,
     };
 
-    let preprocessed_functions = vecmap(contract.functions, |func| PreprocessedContractFunction {
```

Comment:
can you grep for `preprocessed` too so we can also rename variables like `preprocessed_functions` and `preprocessed_contract`?

Reply:
done!